GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+
+## Community Roles and Responsibilities
+
+Refer to [Community Membership](/community-membership.md).
+
+## Community Structure
+
+![community organization](/media/governance/community_organization.png)
+
+The project is comprised of and operated by the following types of subgroups:
+
+- Project Management Committee, PMC
+- Maintainers
+- Organizer Committee
+- Developer Group
+    - Special Interest Groups, SIGs
+    - Working Groups, WGs
+
+### Project Management Committee
+
+The PMC functions as the core management team that oversees the TiKV community. The PMC has additional responsibilities over and above those of Maintainers. These responsibilities ensure the smooth running of the project. PMC members are expected to review code contributions, participate in strategic planning, approve changes to the governance model and manage the copyrights within the project outputs.
+
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Maintainers or Committers, and makes all major decisions for the future with respect to TiKV, such as project-level governance policies, management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives.
+
+Membership of the PMC is by invitation from the existing PMC members. A nomination will result in discussion and then a vote by the existing PMC members. PMC membership votes are subject to consensus approval of the current PMC members.
+
+Given that PingCAP has built and supported this project in the first place, they reserve the right to appoint and form the first committee for a period of first year from the time this document becomes effective. Future PMC numbers are generated based on the rules described above.
+
+The number of PMC members should be limited to 6. This number is chosen to ensure that sufficient points of view are represented, while preserving the efficiency of the decision making process. For a list of the current PMC members, see the [PMC Members](/committee/.README.md#members).
+
+<!--- need the link to a Guiding Principals (Missions, Values; to be added separately) page as the principals for PMC -->
+
+### Maintainers
+
+While the PMC is the core management body that oversees the project, Maintainers are the technical authority in the Developer Group who function as planners and designers of the project. Maintainers set technical direction and priorities for the sub-project, and ensure its continued health and development. They make or approve design decisions for the project - either directly or by delegating these responsibilities. Maintainers are appointed by the PMC with anonymous votes.
```

Comment:
> ...core management body that oversees...

The word "body" looks weird to me. Maybe person is better?

Reply:
It's not a person, more of a group of people.

Reply:
"body" seems more like a biological word to me. Can you show me similar usages?

Reply:
"A statutory authority is a body set up by law which is authorised to enact legislation on behalf of the relevant country or state." - https://en.wikipedia.org/wiki/Statutory_authority